First/src/firstPackage/EqualOccurances.java

```diff
@@ -0,0 +1,35 @@
+package firstPackage.First.src.firstPackage;
+
+public class EqualOccurances {
+    public static boolean equalNumberOccurances(String input) {
+        boolean output = true;
```

Comment:
Flag default should always be false. Please re-write logic considering flag value false.

Reply:
Done ! Changed the logic of program